src/yunohost/settings.py

```diff
@@ -299,3 +300,13 @@ def reconfigure_ssh(setting_name, old_value, new_value):
 def reconfigure_ssh(setting_name, old_value, new_value):
     if old_value != new_value:
         service_regen_conf(names=['postfix'])
+
+@post_change_hook("pop3.enabled")
+def reconfigure_dovecot(setting_name, old_value, new_value):
+    if old_value != new_value:
+        service_regen_conf(names=['dovecot'])
+
+    if new_value is True:
+        subprocess.call(['ynh_package_install', 'dovecot-pop3d'])
```

Comment:
@alexAubin, I'm not sure if this is the right approach?

I suppose I should also try to `try/catch` here too?